src/meltano/cli/run.py

```diff
@@ -112,9 +112,13 @@ async def run(
         if validate_block_sets(logger, parsed_blocks):
             logger.debug("All ExtractLoadBlocks validated, starting execution.")
         else:
-            tracker.track_command_event(cli_tracking.FAILED)
+            tracker.track_command_event(cli_tracking.ABORTED)
             raise CliError("Some ExtractLoadBlocks set failed validation.")
-        await _run_blocks(tracker, parsed_blocks, dry_run=dry_run)
+        try:
+            await _run_blocks(tracker, parsed_blocks, dry_run=dry_run)
+        except Exception as err:
+            tracker.track_command_event(cli_tracking.FAILED)
+            raise err
```

Comment:
@pnadolny13 in the vast majority of cases, you should now always see a tracking event signaling a CLI failure for `meltano run`. If a pre-flight check of the ExtractLoadBlock fail's I send an `aborted` instead. 

Would take an uncaught exception during validation or much earlier issue for that event to go missing.